io_scene_godot/converters/armature.py

```diff
@@ -46,29 +45,37 @@ def __init__(self, bone_name, parent_name):
         self.pose = mathutils.Matrix()
 
 
-def export_bone(pose_bone, exclude_ctrl_bone):
+def should_export(export_settings, armature_obj, rest_bone):
```

Comment:
I prefer it if functions that evaluate to booleans have a consistant "order". Either they return True at the end, and all conditionals inside evaluate to False, or they return False at the end and all conditional evaluate to True. This may make individual conditionals harder to read, but helps to prevent debugging issues due to conditional order.

So in this case, I would invert the `export_settings['use_control_bone']` line such that it returns "True" if it needs to be exported, and change the final line to `return False`

Reply:
make sense